config/config.schema.json

```diff
@@ -1028,6 +1028,13 @@
         "manage"
       ]
     },
+    "batchProcessesDryRun": {
+      "type": "boolean",
```

Comment:
If we define this as a boolean, what happens if I don't provide the attribute at all in config.yaml?